components/Blocks/PlainHero.js

```diff
@@ -1,10 +1,10 @@
-import React, {PropTypes} from 'react';
+import React, { PropTypes } from 'react';
```

Comment:
And no, I didn't fix all of these manually! `eslint` has a `--fix` argument you can use on the CLI that fixes easy problems.